src/staticanalysis/bot/static_analysis_bot/workflow.py

```diff
@@ -276,74 +259,18 @@ def detect_issues(self, analyzers, revision):
             analyzer = analyzer_class()
 
             # Run analyzer on revision and store generated issues
-            issues += analyzer.run(revision)
+            analyser_issues = analyzer.run(revision)
+            issues += analyser_issues
 
         return issues
 
-    def build_improvement_patch(self, revision, issues):
-        '''
-        Build a Diff to improve this revision (styling from clang-format)
-        '''
-        assert isinstance(issues, set)
-
-        # Only use publishable issues
-        # and sort them by filename
-        issues = sorted(
-            filter(lambda i: i.is_publishable(), issues),
-            key=lambda i: i.path,
-        )
-
-        # Apply a patch on each modified file
-        for filename, file_issues in itertools.groupby(issues, lambda i: i.path):
-            full_path = os.path.join(settings.repo_dir, filename)
-            assert os.path.exists(full_path), \
-                'Modified file not found {}'.format(full_path)
-
-            # Build raw "ed" patch
-            patch = '\n'.join(filter(None, [issue.as_diff() for issue in file_issues]))
-            if not patch:
-                continue
-
-            # Apply patch on repository file
-            with build_temp_file(patch, '.diff') as patch_path:
```

Comment:
You're not trying to apply the patch to verify it works anymore, why?

Reply:
It's already applied in fact, as `clang-format` patches directly the file and we get the patch by using `hg diff`

Reply:
Sounds good then!